pkg/client/client.go

```diff
@@ -46,12 +46,13 @@ type VersionRange struct {
 }
 
 type Client struct {
-	conn *grpc.ClientConn
-	fs   pb.FsClient
+	conn  *grpc.ClientConn
+	fs    pb.FsClient
+	cache pb.CacheClient
 }
 
 func NewClientConn(conn *grpc.ClientConn) *Client {
-	return &Client{conn: conn, fs: pb.NewFsClient(conn)}
+	return &Client{conn: conn, fs: pb.NewFsClient(conn), cache: pb.NewCacheClient(conn)}
```

Comment:
As I was playing around with this code this ended up being a bit weird.

We are using a single GRPC connection to talk both to the DL server and to the Cached server.

But in reality if you create a `NewClientConn` with a GRPC connection the the server, the cached client commands won't work and vice versa.

I'm going to refactor this a bit.